agent/acl.go

```diff
@@ -2,13 +2,11 @@ package agent
 
 import (
 	"fmt"
-
-	"github.com/hashicorp/serf/serf"
-
 	"github.com/hashicorp/consul/acl"
 	"github.com/hashicorp/consul/agent/structs"
 	"github.com/hashicorp/consul/api"
 	"github.com/hashicorp/consul/types"
+	"github.com/hashicorp/serf/serf"
```

Comment:
nit: We conventionally group packages with `go imports -local "github.com/hashicorp/consul"`.